lib/std/build.zig

```diff
@@ -242,6 +242,13 @@ pub const Builder = struct {
             null;
     }
 
+    /// Create the given object with the builder allocator
+    pub fn create(self: Builder, obj: anytype) *@TypeOf(obj) {
```

Comment:
obj: anytype makes me nervous because you really should be telling the allocator exactly what type you want to allocate, and then we would want the initialization value to coerce to it. But there's a reason that Allocator does not accept an initialization value - it used to but then we realized that a common use case was to write data into place rather than copying it.

Reply:
I would have thought RLS and "automatic parameter references" would elide any copy in this case?